feature/bgp/policybase/otg_tests/route_installation_test/route_installation_test.go

```diff
@@ -528,6 +528,11 @@ func TestEstablish(t *testing.T) {
 	t.Logf("Start DUT interface Config")
 	configureDUT(t, dut)
 
+	// Configure Network instance type on DUT
+	t.Log("Configure Network Instance type")
+	dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
+	gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
```

Comment:
Why is this section different between ATE and OTG versions of the script. Below from the ATE

// Configure Network instance type on DUT
	t.Log("Configure Network Instance type")
	dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
	dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)